core/src/main/java/feast/core/config/SecurityConfig.java

```diff
@@ -61,9 +61,9 @@ AuthenticationManager authenticationManager() {
 
     if (securityProperties.getAuthentication().isEnabled()) {
       switch (securityProperties.getAuthentication().getProvider()) {
-        case "GoogleOpenID":
+        case "jwt":
           providers.add(
-              new GoogleOpenIDAuthenticationProvider(
+              new DefaultJwtAuthenticationProvider(
```

Comment:
DI didn't work?

Reply:
it did but wasn't simpler.  It didn't seem to have value as I expected.